### ISSUE TITLE
Change AS verb "observe" to "query"

### ISSUE THREAD

**Issue by raucao** [sockethub/sockethub#647]
> I'm proposing to change it to "query" instead, because the term 1) makes more sense in the language itself and 2) is better suited for many more query types.

Background is adding query types for room info (#540) and other things that are not room attendance. And since the next release will break AS object notations, this would be a good time to introduce more breaking changes all at once to have avoid more breakage in the future.

**Comment by silverbucket** [comment]
I definitely think it sounds better than observe :) can you provide some more examples of types of actions this could apply to? For example, so far we just have:

* attendance

But I could think of it as something which applies to any non-human aspect of the platform? eg. NickServ or an XEP to get info about a room, attendance or a topic.

Which brings up the fact that topic already exists as a type of `update`,  (you can update your user address/change nick, and you can update the topic of a room). That could be fine I guess, if you imagine reading a history you would see someone `update` the `topic`, and someone `query` the `topic`. When someone changes the topic Sockethub sends an `update` to other users (IRC platform).

**Comment by raucao** [comment]
Yes, exactly what you described. And as stated above, we need a query type for room info, as well as other things for mostly XMPP at the moment. For example querying the room list from a server. But really, just querying any information from the other side would be covered by it.
